waspc/waspls/src/Wasp/LSP/Prisma/Util.hs

```diff
@@ -0,0 +1,9 @@
+module Wasp.LSP.Prisma.Util where
+
+import qualified Wasp.Psl.Ast.Schema as Psl.Ast
+
+showModels :: Psl.Ast.Schema -> String
```

Comment:
```suggestion
showModelNames :: Psl.Ast.Schema -> String
```
this maybe? Or you on purpose wanted to keep it like `showModels` and the fact that you are showing names is an implementation detail that might change?